multiversx_sdk_cli/transactions.py

```diff
@@ -211,8 +203,8 @@ def tx_to_dictionary_as_inner_for_relayed_V1(tx: Transaction) -> Dict[str, Any]:
     dictionary: Dict[str, Any] = {}
 
     dictionary["nonce"] = tx.nonce
-    dictionary["sender"] = base64.b64encode(Address.new_from_bech32(tx.sender).get_public_key()).decode()
-    dictionary["receiver"] = base64.b64encode(Address.new_from_bech32(tx.receiver).get_public_key()).decode()
+    dictionary["sender"] = base64.b64encode(tx.sender.get_public_key()).decode()
```

Comment:
Let's remove `tx_to_dictionary_as_inner_for_relayed_V1` and legacy relayed V1 support in v10 :crossed_fingers:

Reply:
we will.